rules.adoc

```diff
@@ -1045,21 +1045,27 @@ used.
 All teams qualified to the international competition *must* share their
 designs, both hardware and software, with all present and future participants.
 These teams are also required to send a digital portfolio before the
-competition. Further details on how will be provided by the Organizational
+competition. Further details on how will be provided by the Organization
 Committee.
 
 During the competition days of the International Competition (as well as before
 the event) the team members are responsible for checking all relevant
-information published by the Soccer Organizational Committee, General Chairs,
+information published by the Soccer Organization Committee, General Chairs,
 or any other RoboCup official.
 
 There will also be a SuperTeam competition, in which various people from around
 the world share their robots in one "SuperTeam" and play against other
-SuperTeams on a so called "Big Field". The full rules of this challenge can be found at
+SuperTeams on a so called "Big Field".  The full rules of this challenge can be found at
 https://robocupjuniortc.github.io/soccer-rules/master/superteam_rules.html
 
+NOTE: {++To make the communication between multiple robots in a single
+"SuperTeam" easier and the games more manageable, the Organization Committee
+will provide each team a communication module. Each team will be expected to
+interface with this module using a single 2.54mm GPIO pin. More details will be
+provided by the Organization Committee shortly before the competition.++}
+
```

Comment:
This is only a "will" for the international level and much more of a "may" for local/regional/superregional levels.

Reply:
Agreed, but this is in the "international" section after all :)

Reply:
As it is it's not really making communication easier - that I think starts at UART or something like that. At the moment it only makes start/stop control easier.

Reply:
Agreed on intl. section, did not spot that in the diff...

Reply:
That's fair -- I'd be happy to take any suggestions you might have here. I was taking a bit longer view here which might not be there yet :slightly_smiling_face:

Reply:
{++To make SuperTeam games more manageable now and make communication between multiple robots in a SuperTeam easier in the future the Organization Committee will provide each team with a communication module. Each team will be expected to interface with this module using a single 2.54mm GPIO pin now and the Organiyation committee plans on extending this to using UART for more complex applications in future years. More details will be provided by the Organization Committee shortly before the competition.++}

Reply:
I don't know if that makes the the sentence overly complicated but I'd like to provide as much "look forward" as to increase what ideas teams can generate and contribute in the future. Feel free to simplify, I'm not very good at changing existing phrasings.

Reply:
Thanks @dbscoach!

Would you mind taking another look?